src/view/com/util/layouts/LoggedOutLayout.tsx

```diff
@@ -73,9 +72,7 @@ export const LoggedOutLayout = ({
             contentContainerStyle={styles.scrollViewContentContainer}
             keyboardShouldPersistTaps="handled"
             keyboardDismissMode="on-drag">
-            <View style={[styles.contentWrapper, isWeb && a.my_auto]}>
-              {children}
-            </View>
+            <View style={[styles.contentWrapper]}>{children}</View>
```

Comment:
Should consider making this a KeyboardAvoidingView or something

Reply:
Agree on this. We actually should probably use one of the native libraries for this like the one you used in Graysky or the other popular one. `KeyboardAvoidingView` can be a bit janky in general especially on Android.